tests/operations.rs

```diff
@@ -383,3 +383,77 @@ fn addmod_with_zero_denominator() {
     ];
     run_program_assert_result(program, 0);
 }
+
+#[test]
+fn test_sgt_unsigned_greater_than() {
```

Comment:
Nit: the numbers are still interpreted as signed.
```suggestion
fn test_sgt_positive_greater_than() {
```

Reply:
Similar for later uses.